src/lib.rs

```diff
@@ -848,4 +844,26 @@ mod tests {
             let _  = rx.recv();
         }
     }
+
+    #[test]
+    fn test_repeat_overlap() {
```

Comment:
So what property does this test check, exactly?

Reply:
I rename it to be more explicit and I add some comment:
```rust
#[test]                                                                  
fn test_overload() {                                                     
    // Make sure that if timer is overload we still can communicate with 
    // and ask it to stop or schedule an other task.                     
    // An overloaded timer is easily done by scheduling a task longer to 
    // execute than it repeat interval.                                  
```